homework_03/06_vladislav_kolev/circle.py

```diff
@@ -0,0 +1,39 @@
+from enum import Enum, auto
+from math import sqrt, pow
+
+class RelativePosition(Enum):
+    NO_COMMON_POINTS = 1
+    TOUCHING = 2
+    INTERSECTING = 3
+    SAME = 4
+
+class Point:
```

Comment:
OK

Reply:
i sugest use other value for enum
0 - no_common_points
1-touching
2-intersecting
3-same